packages/react-text/src/stories/TextBestPractices.md

```diff
@@ -0,0 +1,7 @@
+## Best Practices
+
+### Do
+
+- Use Text whenever you need some stylized text
+- Use Text to display read-only text
```

Comment:
I believe this is important to point out, so that users will avoid making Text as link, button so smth like that. Why do think this should be removed?

Reply:
Oh, okay, I think I just misunderstood it. Do you think that maybe be clearer to add a Don’t section with "use Text for a link, button or other actionable text"?

Reply:
IMO it is repetitive, as the "read-only text" it clearly means that. I will also add @andrefcdias for his opinion on this.

Reply:
I'll go ahead with merging this, if needed I will create another PR for any improvements on the docs for best practices.